Source/SwiftLintFramework/Rules/MarkRule.swift

```diff
@@ -35,10 +35,7 @@ public struct MarkRule: ConfigurationProviderRule {
         let options = ["MARK:[^ ]", "[^ ]MARK: [^-]", "\\sMARK:[^ ]", "MARK:[ ][-][^ ]"]
         let pattern = "(" + options.joinWithSeparator("|") + ")"
 
-        return file.matchPattern(pattern).flatMap { range, syntaxKinds in
```

Comment:
@jpsim I based this on `TodoRule.swift` - `isCommentLike` seems like a 'wider' check than just `.Comment`.
I understand that `withSyntaxKinds` is better approach, but maybe other values from `isCommentLike` should also be included?